R/mobr_boxplots.R

```diff
@@ -55,7 +55,9 @@ calc_biodiv_single_group <- function(abund_vec, n_rare){
    
    out_vec <- c("N"       = NA,
                 "S"       = NA,
-                "S_rare"  = NA,
+                "S_rare1"  = NA,
+                "S_rare2"  = NA,
+                "S_rare3"  = NA,
                 "S_asymp" = NA,
                 "PIE"     = NA,
```

Comment:
Can you please add a code comment or rename the `S_rare#` objects so that it is clearer what they refer to.